mlflow/models/signature.py

```diff
@@ -56,7 +56,7 @@ class ModelSignature:
     :py:class:`Schema <mlflow.types.Schema>`.
     """
 
-    def __init__(self, inputs: Schema, outputs: Schema = None):
+    def __init__(self, inputs: Schema, outputs: Schema = None, parameters: ParamSchema = None):
```

Comment:
I think this should be `params` instead of `parameters`. We should replace all instances of `parameters` with `params`